src/status_im/ethereum/stateofus.cljs

```diff
@@ -4,8 +4,16 @@
 
 (def domain "stateofus.eth")
 
-(defn subdomain [username]
-  (str username "." domain))
+(defn subdomain
+  "checks if the username is a status username or a ens name
+  for that we check if there is a dot in the username, which
+  would indicate that there is already a domain name so we don't
+  concatenated stateofus domain to it"
+  [username]
+  (if (and username
```

Comment:
What happens if it's nil? is it a valid value?
looks like if it's nil we would do:
`(str nil ".domain")`
Is that a valid case?

Otherwise I would have the check on top
`(when (seq username)...)`